app/models/match_routes/eleven.rb

```diff
@@ -0,0 +1,75 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module MatchRoutes
+  class Eleven < Base
+    def title
+      Translation.translate('Match Route Eleven')
+    end
+
+    def self.available_sub_types_for_search
+      match_steps_for_reporting.keys
+    end
+
+    def self.match_steps
+      {
+        'MatchDecisions::Eleven::ElevenHsaAcknowledgesReceipt' => 1,
+        'MatchDecisions::Eleven::ElevenHsaAcceptsClient' => 2,
+      }
+    end
+
+    def self.match_steps_for_reporting
+      {
+        'MatchDecisions::Eleven::ElevenHsaAcknowledgesReceipt' => 1,
+        'MatchDecisions::Eleven::ElevenHsaAcceptsClient' => 2,
+        'MatchDecisions::Eleven::ElevenConfirmHsaAcceptsClientDeclineDndStaff' => 2,
+      }
+    end
+
+    def required_contact_types
+      [
+        'dnd_staff',
+        'housing_subsidy_admin',
```

Comment:
Should these be route specific?

Reply:
probably not, just checking.

Reply:
From what I am seeing, these are not route-specific in any of the other routes. Elsewhere, we are overriding the label when displayed using reoute-specific translations.